threshsign/src/CMakeLists.txt

```diff
@@ -10,13 +10,16 @@ set(common_source_files
 add_library(threshsign STATIC ${common_source_files})
 target_include_directories(threshsign PUBLIC ../include)
 target_link_libraries(threshsign PUBLIC Threads::Threads)
+find_library(CRYPTO_STATIC_LIBRARY NAMES libcrypto.a PATHS ${OPENSSL_ROOT_DIR}/lib/ REQUIRED)
```

Comment:
use `find_package(OpenSSL)` there should be a variable populated for static library

Reply:
done